cli/dts/lib.deno.unstable.d.ts

```diff
@@ -346,9 +341,14 @@ declare namespace Deno {
     | "f64"
     | "pointer";
 
+  type NativeParameterType =
```

Comment:
I was thinking, maybe it makes things more explicit if we had `NativeParameterType` and `NativeReturnType`, as in:
* `NativeParameterType` can be sent from JS to C
* `NativeReturnType` can be sent from C to JS

Of course in case of callbacks the types would be reversed: parameters would be `NativeReturnType` and the return value would have type `NativeParameterType`.

So maybe we should consider`ToNativeType` and `FromNativeType` or something.

Reply:
I agree, and do plan on doing a rework on the FFI TS types. f.ex. Currently `NativeType` is the "base type" and `NativeNumberType` is a narrowed down type from that. I will do a follow-up to rework the types into something that hopefully resembles a reasonable whole.

A `ToNativeType` and `FromNativeType` sounds splendid for that.